Makefile

```diff
@@ -87,7 +87,7 @@ deploy-dev: manifests
 	cd config/default && if [ ${KSERVE_ENABLE_SELF_SIGNED_CA} != false ]; then \
 	echo > ../certmanager/certificate.yaml; \
 	else git checkout HEAD -- ../certmanager/certificate.yaml; fi;
-	kubectl apply --server-side=true -k config/overlays/development
+	kubectl apply --server-side=true --force-conflicts -k config/overlays/development
```

Comment:
Do we want `--force-conflicts` on the other 3 `kubectl apply` commands in `deploy-dev` and `deploy` as well?

Reply:
Need to take a deeper look, when I run locally I had to add this flag to apply the change